src/app/services/project_service.js

```diff
@@ -228,7 +228,13 @@ angular
                     }
                     var node = project.nodes[model];
                     var column = _.find(node.columns, function(col, col_name) {
-                        return col_name.toLowerCase() == test_column.toLowerCase();
+                        // strip quotes from start and end of test column if present in both locations
+                        // this is necessary to attach a test to a column when `quote: true` is set for a column
+                        let test_column_name = test_column;
+                        if (test_column.startsWith('"') && test_column.endsWith('"')) {
```

Comment:
Quote characters are different on different adapters. E.g. on `dbt-bigquery` it will be:
```json
  "column_name": "`id`",
```

It is less common to quote columns on BQ, since it doesn't generally support spaces / special characters — but it does still support reserved keywords (e.g. `from`) as column names if quoted.

I'm not sure if there's a cleverer regex check we could/should be doing here instead of hard-coding `"`

Reply:
Just pushed another commit here. I tried to go the regex route, but i think it was a little too complicated. The specific problem is that I wanted to ensure that the start and end quote chars are the same (eg. we shouldn't strip quotes from the string `` "some_column` ``). I ended up enumerating different quote chars based on data platform in the javascript. I made the backtick BQ-only, which is not correct, but wondering if it's close enough?

Other thoughts:
- I am very open to other suggestions/ideas on how to handle this, or just a statement that "this ain't it" and i'll have a harder think about it :)
- Can you confirm my assumption that the `metadata.adapter_type` field will read "bigquery" on bigquery?
- Really reinforcing my desire for some sort of adapter-specific info within the manifest. That could look like including quote chars in the `metadata` blob to avoid needing to encode this sort of info in js!

Reply:
`spark` & `databricks` also use backticks as quotes :)

Agree this isn't _really_ scalable to other adapters! It could make sense to include this kind of "static" adapter-specific info somewhere in the manifest — though if I had to pick between doing that, and including the original unquoted `column_name` value on the test node, I have a slight preference for the latter

Reply:
at the end of the day - I'm fine with this approach for the time being

Reply:
ok - just added the same logic for spark + databricks and a test too! 🏓